[tiny PR] minor update to support section

manmeetkaur opened the pull request:
This section seems to be confusing with repetition, trying to fix it. 
<img width="998" alt="Screenshot 2023-07-10 at 7 57 37 PM" src="https://github.com/astronomer/docs/assets/75730393/d1a3753f-8104-4b5e-bda9-35a6ba06d5c9">

jwitz commented on astro/astro-support.md:
```diff
@@ -89,9 +89,10 @@ If you've already copied task logs or Airflow component logs, send them as a par
 
 Astronomer recommends that you submit support requests in the Cloud UI. If you can't access the Cloud UI, you can submit support requests on the [Astronomer support portal](https://support.astronomer.io).
```
```suggestion
Astronomer recommends that you submit support requests in the Cloud UI. If you can't access the Cloud UI, sign in to the [Astronomer support portal](https://support.astronomer.io) and create a new support request. 
```

jwitz commented on astro/astro-support.md:
```diff
@@ -89,9 +89,10 @@ If you've already copied task logs or Airflow component logs, send them as a par
-If you're new to Astronomer, you'll need to create an account on the Astronomer support portal to submit a support request. Astronomer recommends that you use the same email address that you use to access Astro for your account. If you're working with a team and want to view support tickets created by other team members, use your work email or the domain you share with other team members for your ...
-
-If you're an existing customer, sign in to the [Astronomer support portal](https://support.astronomer.io) and create a new support request.
+- If you're an existing customer, sign in to the [Astronomer support portal](https://support.astronomer.io) and create a new support request.
+- If you're new to Astronomer, you'll need to create an account on the Astronomer support portal to submit a support request. Astronomer recommends that you use the same email address that you use to access Astro for your account. 
+- If you're working with a team and want to view support tickets created by other team members, use your work email or the domain you share with other team members for your account (for example, `@astronomer.io`).
+- If your team uses more than one email domain, contact Astronomer and ask to have the additional domains added to your organization.
```
```suggestion
If you're new to Astronomer, you'll need to create an account on the Astronomer support portal to submit a support request. Astronomer recommends that you create an account with the same email address that you use to access Astro. This allows you to view support tickets from other team members that have email addresses with the same domain.

If your team uses more than one email domain, add all domains to your Organization so that team members with different email domains can view each others' support requests. See [Create and manage domains for your Organization](manage-domains.md).
```

jwitz reviewed (approved):
@manmeetkaur With small changes like these, it's helpful to include a brief description in the PR so that I know why we're making the change. I still don't quite get why we're making this change, but did my best to edit it. 

Bulleted lists are mainly helpful for listing ideas that are parallel to each other, but don't depend on each other. The contents in the bullet points are all related and build on each other, so I think it's better to format them as paragraphs. 

I changed a lot here, so tag me back if I changed a technical meaning by accident. Otherwise this LGTM

manmeetkaur commented:
> @manmeetkaur With small changes like these, it's helpful to include a brief description in the PR so that I know why we're making the change. I still don't quite get why we're making this change, but did my best to edit it.
> 
> Bulleted lists are mainly helpful for listing ideas that are parallel to each other, but don't depend on each other. The contents in the bullet points are all related and build on each other, so I think it's better to format them as paragraphs.
> 
> I changed a lot here, so tag me back if I changed a technical meaning by accident. Otherwise this LGTM

@jwitz Apologies for not writing the description, I thought it was self-explanatory. The original section, had too many 1-liners which were not grouped together based on the logically. I just happened to be on that page and did not like it :) 

noted about the bullet points - 2 paras look nice! 

Thanks!

jwitz commented:
@manmeetkaur All good, the description can even be as simple as "just cleaning this up" or something similar, just to provide context
